libs/application/templates/general-fishing-license/src/fields/components/ShipInformation/index.tsx

```diff
@@ -56,23 +56,23 @@ export const ShipInformation: FC<ShipInformationProps> = ({
       >
         {name}
       </Text>
-      {registrationNumber && (
+      {!!registrationNumber && (
         <ValueLine
           label={formatMessage(shipSelection.labels.shipNumber)}
           value={registrationNumber.toString()}
           disabled={isExpired}
           color={isExpired ? 'grey' : 'black'}
         />
       )}
-      {grossTons && (
+      {!!grossTons && (
         <ValueLine
           label={formatMessage(shipSelection.labels.grossTonn)}
           value={grossTons.toString()}
           disabled={isExpired}
           color={isExpired ? 'grey' : 'black'}
         />
       )}
-      {length && (
+      {!!length && (
```

Comment:
Why did you go with throwing everything to truthy values?

Reply:
Because sometimes some values returned 0 and then I would only see 0.